web/helpers/string.helper.ts

```diff
@@ -120,7 +120,7 @@ export const getFirstCharacters = (str: string) => {
  */
 
 export const sanitizeHTML = (htmlString: string) => {
-  const sanitizedText = DOMPurify.sanitize(htmlString, { ALLOWED_TAGS: [] }); // sanitize the string to remove all HTML tags
+  const sanitizedText = DOMPurify.sanitize(htmlString, { ALLOWED_TAGS: ["mention-component"] }); // sanitize the string to remove all HTML tags
```

Comment:
We don't need this here since we are stripping the <Mention> before we call the sanitizeHTML